README.md

```diff
@@ -1939,8 +1976,8 @@ Other Style Guides
     }
     ```
 
-  <a name="comments--todo"></a><a name="17.5"></a>
-  - [17.5](#comments--todo) Use `// TODO:` to annotate solutions to problems.
+  <a name="comments--todo"></a><a name="17.6"></a>
```

Comment:
No problem! The numbers that are user-visible can be renumbered at will, should be sorted, and should be added with new sections.

The numbers that are solely used for permalinks (in the `<a name=`) should remain the same forever, and should not be added to new sections.